docs/patterns/messaging-bridge-content.md

```diff
@@ -0,0 +1,88 @@
+- Connect geo-distributed systems when Internet connection is not stable
+- Migrate a single distributed system from one messaging infrastructure to another without the need to stop and re-deploy the whole system
+
+This pattern might not be suitable if:
+
+- At least one of the systems involved relies on specific feature of one messaging infrastructure not present in the other
+- Integration is synchronous in nature (the initiating system requires immediate response)
+- Integration has specific functional or non-functional requirements (e.g. security or privacy concerns)
+- The volume of data for the integration exceeds the capacity of the messaging system or makes messaging not a cost-effective solution to the problem
+
+## Example
+
+There is a legacy application written in .NET Framework for managing production site hosted on site. The application is well-structured with separate components communicating via Microsoft Message Queueing (MSMQ). The application works just fine and the company has no intention of re-writing it, but the IT strategy calls for building new software as cloud-native applications in order to optimize the costs and delivery time.
+
+Single the asynchronous queue-based architecture worked well for the company in the past, the new components are going to use the same architectural approach but with modern technology -- Azure ServiceBus. The implementation does not want to introduce new concepts, such as REST/HTTP communication, just to integrate the two sides and would rather take advantage of the existing mechanisms.
+
+The decision is to use the Messaging Bridge pattern to connect the two systems. It consists of two parts. One part receives messages from a MSMQ queue and forwards them to Azure ServiceBus and the other does the opposite -- takes messages from the Azure ServiceBus and forwards to a destination MSMQ queue.
+
+![Messaging bridge integrating MSMQ and Azure ServiceBus](_images/messaging-bridge-sample.png)
+
+By taking this approach the implementation team can utilize existing infrastructure in the legacy appication to integrate with the new components. The application is not even aware of the fact that the new components are hosted in Azure. Similarly, the new components can communicate with the legacy application in the same way as they communicate between themselves -- by sending Azure ServiceBus messages. The bridge takes care of the forwarding.
+
+## Next steps
+
+The following information may be relevant when implementing this pattern:
+
+- [Competing Consumers](../../patterns/ccompeting-consumers.yml) should be used to ensure the implementation of the Messaging Bridge can cope with the load.
+- [Retry](../../patterns/retry.yml) lets the Messaging Bridge handle transient failures.
+- [Circuit breaker](../../patterns/circuit-breaker.yml) can be used to conserve resources when either side of the bridge is experiencing downtime.
+
+## External links
+
```

Comment:
Maybe a link to: https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingBridge.html#:~:text=The%20Messaging%20Bridge%20is%20a,a%20pair%20of%20corresponding%20channels.